Cargo.toml

```diff
@@ -12,4 +12,4 @@ keywords = ["text-processing", "search", "spelling"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-quickcheck = "^0.4.1"
+quickcheck = "0.6"
```

Comment:
@ignatenkobrain - Is there a specific reason you switched to fixed version spec?

Reply:
@febeling it's not fixed version. Not specifying OP means `^`. so `0.6` == `^0.6`

Reply:
@ignatenkobrain cool, sounds good!